src/components/EmployeeList.jsx

```diff
@@ -1,9 +1,21 @@
+import styled from '@emotion/styled';
```

Comment:
import { styled } from '@mui/material/styles';
https://mui.com/material-ui/customization/how-to-customize/#2-reusable-component